packages/contractkit/src/wrappers/MultiSig.ts

```diff
@@ -0,0 +1,28 @@
+import { Address } from '../base'
+import { MultiSig } from '../generated/types/MultiSig'
+import { BaseWrapper, proxyCall, proxySend, stringToBytes } from './BaseWrapper'
+
+export type SubmitTransactionsParams = Parameters<MultiSig['methods']['submitTransaction']>
+export const submitTransactionsParams = (
+  destination: Address,
+  transaction: string
+): SubmitTransactionsParams => {
+  return [destination, 0, stringToBytes(transaction)]
+}
+
+/**
+ * Contract for handling multisig actions
+ */
+export class MultiSigWrapper extends BaseWrapper<MultiSig> {
```

Comment:
I think this is missing the ability to confirm transactions. For example, in a 2/2 multisig for the spender, for example, one signatory would need to submit the tx and the other one would need to confirm it.

For usability, you could imagine a `submitOrConfirmTransaction` function in this wrapper, that takes a transaction object, and iterates through the multisig transactions from latest tx ID to earliest, looking for an unexecuted transaction that matches the provided parameters. If a match is found, it calls `confirmTransaction`, and if not, it calls `submitTransaction`.

Wdyt?

Reply:
I see what you mean...the second person would just end up adding a new identical reaction to the list rather than confirming in this implementation.  

Iterating through each submitted tx doesn't seem super efficient, but I can't think of another option off the top of my head.  I'll dig in a little.